diagnostic_aggregator/include/diagnostic_aggregator/aggregator.hpp

```diff
@@ -36,29 +36,31 @@
  *\author Kevin Watts
  */
 
-#ifndef DIAGNOSTIC_AGGREGATOR_H
-#define DIAGNOSTIC_AGGREGATOR_H
+#ifndef DIAGNOSTIC_AGGREGATOR__AGGREGATOR_HPP
+#define DIAGNOSTIC_AGGREGATOR__AGGREGATOR_HPP
 
-#include <ros/ros.h>
 #include <string>
 #include <map>
 #include <vector>
 #include <set>
-#include <boost/shared_ptr.hpp>
+#include <memory>
 #include <boost/thread/mutex.hpp>
```

Comment:
can that be replaced with a `std::mutex`?

Reply:
Yes, we can! ;)